src/ServiceControl.MultiInstance.AcceptanceTests/Monitoring/When_a_message_is_imported_twice.cs

```diff
@@ -55,7 +55,7 @@ public class Receiver : EndpointConfigurationBuilder
         {
             public Receiver()
             {
-                EndpointSetup<DefaultServerWithAudit>(c => c.ForwardReceivedMessagesTo("audit"));
+                EndpointSetup<DefaultServerWithAudit>(c => c.AuditProcessedMessagesTo("audit"));
```

Comment:
This does not do what we need for the test. The `DefaultServerWithAudit` endpoint setup already enables auditing and then we override the config and enable auditing again. The result is that the message is processed once and then we make assertions about it's behavior when it's imported twice.

What we need to do is somehow make sure that the messages makes it into the audit queue twice. I wrote a small behavior to do it in the smoke test PR https://github.com/Particular/ServiceControl/pull/2096/files#diff-a848910d5014bb6804422d013909449aR60-R69